growbox/systime_settings.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+
+def set_systime(date_time):
+    os.system("sudo date --set='" + date_time + "'")
```

Comment:
:thinking: what if I pass `date_time="42'; sudo rm -rf /; echo '"` here? :boom: 

And that can be quite easily done by sending malicious requests to the time-setting form. Always escape shell & SQL arguments. In this case use [`subprocess.run`](https://docs.python.org/3/library/subprocess.html#module-subprocess). It will escape everything for you

Reply:
Fix it in the next commit.